spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -310,4 +342,37 @@ def then_I_see_a_page_reverted_success_notice
   def and_I_see_I_saved_it_last
     expect(page).to have_content("Last saved by Test author")
   end
+
+  def and_I_visit_the_scheduling_page
+    visit step_by_step_page_schedule_path(@step_by_step_page)
+  end
+
+  def when_I_visit_the_scheduling_page
+    and_I_visit_the_scheduling_page
+  end
+
+  def and_I_fill_in_the_scheduling_form
+    fill_in 'scheduled_at', with: '2030-04-20 10:26:51 UTC'
+  end
+
+  def and_I_fill_in_the_scheduling_form_with_a_date_in_the_past
+    fill_in 'scheduled_at', with: '1937-04-20 10:26:51 UTC'
+  end
+
+  def when_I_submit_the_form
+    click_on 'Schedule to publish'
+  end
+
+  def and_there_should_be_no_schedule_button
+    expect(page).not_to have_content 'Schedule to publish'
```

Comment:
Nitpick: You can check that the button itself doesn't exist.
e.g. expect(page).not_to have_css("button", text: "Schedule to publish")

Reply:
I did that yesterday 🤔. Not sure why you can still see `expect(page).not_to have_content 'Schedule to publish'` Could you double check the Files tab, please? I cannot see it there.

Reply:
Ah... I see now. It's still in a previous commit. _Off to fix it_